pkg/policies/policies.go

```diff
@@ -50,6 +50,10 @@ func (e *PolicyError) Error() string {
 	return fmt.Sprintf("policy error: %s", e.err.Error())
 }
 
+func (e *PolicyError) Unwrap() error {
```

Comment:
interesting, what was the result before this?

Reply:
It was losing the error wrapping chain, so it couldn't be recognized as a RPC error with `errors.As`.